lookout/core/metrics.py

```diff
@@ -78,7 +78,7 @@ def _child_samples(self):
         )
 
 
-def submit_event(key: str, value: Union[int, float, bool], description: str = ""):
+def record_event(key: str, value: Union[int, float, bool], description: str = ""):
```

Comment:
Looks like we don't have any tests for this function, otherwise we would have to edit the tests too. This is sad 😢

Reply:
Im fixing this. I forgot to add the files where the function is referenced.

Reply:
Solved